test/routes/src/server/routes/t_login.js

```diff
@@ -0,0 +1,105 @@
+// load server/server.js via proxyquire to stub webpack-assets.json
+import proxyquire from 'proxyquire';
+
+// import login routes for testing
+import loginRoutes from '../../../../../src/server/routes/login';
+
+const routesStub = {
+  login: loginRoutes,
+  api: () => {},
+  github: () => {},
+  githubAuth: () => {},
+  universal: () => {},
+  // tell proxyquire not to try to load stubbed module
+  '@noCallThru': true
+};
+const stubDependencies = {
+  './routes/': routesStub
+};
+
+const app = proxyquire(
```

Comment:
It was me who used proxyquire + routes stub pattern in [t_login.js](https://github.com/canonical-ols/javan-rhino/blob/master/test/routes/t_login.js) but I've seen @katiefenn testing routes in simpler and more isolated way, where we just imports the route to test and create an express app with that route only.

Like:
https://github.com/canonical-ols/build.snapcraft.io/blob/master/test/routes/src/server/routes/t_github.js#L9-L15

It's not that the way with stubbed server here is wrong, but because it starts the whole app server it may 'test' more than it should.